<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaffManager</class>
 <widget class="QWidget" name="StaffManager">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>359</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>人事管理窗口</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>职工管理</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>员工ID</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>姓名</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>部门</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_11">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_9">
           <property name="text">
            <string>显示全部</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" rowspan="2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>录入员工</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>维护信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_12">
           <property name="text">
            <string>删除离职职工</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>保存修改</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>撤销修改</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>进货商管理</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="comboBox_2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_10">
           <property name="text">
            <string>显示全部</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" rowspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>添加进货商</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>删除进货商</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>保存修改</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>撤销修改</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="tableView_2"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
